osu.Framework/Graphics/Video/FFmpegFuncs.cs

```diff
@@ -13,6 +13,11 @@ namespace osu.Framework.Graphics.Video
 {
     public unsafe class FFmpegFuncs
     {
+        static FFmpegFuncs()
+        {
+            DynamicallyLoadedBindings.FunctionResolver = new FFmpegFunctionResolver();
```

Comment:
`FFmpegFuncs` is shared across all platforms but `FFmpegFunctionResolver` crashes on any platform that's not desktop. Noticed this while testing iOS.

```suggestion
            if (RuntimeInfo.IsDesktop)
                DynamicallyLoadedBindings.FunctionResolver = new FFmpegFunctionResolver();
```